src/views/TabView/TabBarBottom.js

```diff
@@ -39,6 +39,7 @@ type Props = {
   getLabelText: (scene: TabScene) => string;
   renderIcon: (scene: TabScene) => React.Element<*>;
   style: any;
+  labelStyle: any?;
```

Comment:
oh, totally missed that.  Sorry.  Fixed in my last commit.  Thanks for the catch.